client/my-sites/plan-features/actions.jsx

```diff
@@ -32,7 +33,7 @@ const PlanFeaturesActions = ( {
 		className
 	);
 
-	if ( current ) {
+	if ( current && ! isInSignup ) {
```

Comment:
I wonder: we need this because a Jetpack site already has a free plan assigned? Because we use the same code in Calypso signup (http://calypso.localhost:3000/start) and it works without having to pass down `isInSignup`.